src/web/lib/getZIndex.tsx

```diff
@@ -62,8 +62,12 @@ const decideIndex = (name: string): number | null => {
 	return decided || null;
 };
 
-export const getZIndex = (zIndex: ZIndex): string =>
-	`z-index: ${decideIndex(zIndex)};`;
+export const getZIndex = (zIndex: ZIndex): string => {
+	const selectedIndex = decideIndex(zIndex);
```

Comment:
as `decideIndex` can return `null` we need to handle that case

Reply:
Is it possible to make it so `decideIndex` isn't able to return `null`?